FunctionalTableData/TableView/FunctionalTableData.swift

```diff
@@ -386,8 +386,8 @@ public class FunctionalTableData: NSObject {
 					update.cellConfig.update(cell: cell, in: tableView)
 					
 					let section = sections[update.index.section]
-					let style = section.mergedStyle(for: update.index.row)
-					style?.configure(cell: cell, in: tableView)
+					let style = section.mergedStyle(for: update.index.row) ?? CellStyle()
```

Comment:
I'd make `mergedStyle` return a non-optional and have it return the `CellStyle()`